cmd/admin-handlers.go

```diff
@@ -1089,7 +1089,7 @@ func (a adminAPIHandlers) ConsoleLogHandler(w http.ResponseWriter, r *http.Reque
 				return
 			}
 			w.(http.Flusher).Flush()
-		case <-GlobalServiceDoneCh:
+		case <-GlobalContext.Done():
```

Comment:
This will catch both server shutdown and a closed connection: 
```suggestion
		case <- ctx.Done():
```